BGL/doc/BGL/Concepts/EdgeListGraph.h

```diff
@@ -0,0 +1,44 @@
+/*!
+\ingroup PkgBGLConcepts
+\cgalConcept
+
+The concept `EdgeListGraph` refines the concept `Graph` and adds
+the requirement for traversal of all edges in a graph.
+
+\cgalRefines `Graph`
+\cgalHasModel `CGAL::Polyhedron_3`
+\cgalHasModel `CGAL::Surface_mesh`
+
+*/
+class EdgeListGraph{};
+
+
+/*! \relates EdgeListGraph
+ * returns an iterator range over all edges.
+ */
+std::pair<boost::graph_traits<EdgeListGraph>::edge_iterator,
```

Comment:
All the free functions of all the concepts are missing `template <class FooBarGraph>`